livestream/.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,82 @@
+name: Docker CI/CD
+
+on:
+  push:
+    branches:
+    - main
```

Comment:
flyby the branch name is now master ;)

Reply:
good catch!

Reply:
This probably would have drove me nuts later on when CI didn't work